app/views/ui-components/v1/cards/_metal_level_select.html.slim

```diff
@@ -358,9 +374,25 @@ javascript:
     planType = options.planType || element.dataset.planType;
     referencePlanID = options.referencePlanID || element.id;
     sponsoredBenefitId = options.sponsoredBenefitId;
+    isStandardPlan = options.isStandardPlan || element.dataset.isStandardPlan;
+    isPvpEligible = options.isPvpEligible || element.dataset.isPvp;
```

Comment:
I think the above 2 lines may now want to be something in pattern of:
```
isStandardPlan = options.hasOwnProperty('isStandardPlan') ? options.isStandardPlan : element.dataset.isStandardPlan;
```
since boolean val of false in `options` would get effectively disregarded (and in the new code flow, the null `element`'s attribute would be dereferenced, raising error).